templates/tuning.yaml.erb

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Wouldn't it be better to have the conditional logic here rather than in the puppet code?  That way you could easily see what the final tuning.yaml is going to look like.